### DISCUSSION TITLE
Can I hide all "node series" that are not currently selected (Not sure how to explain itthe picture i've included should make my question make sense lol)

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1464] [Help]
Hey everyone. So in this picture below, imagine that I have the node series "Prepositional phrases add clutter" selected" Would it then be possible to have everything in the node series "Sentence starting clutter" be hidden, and then vise versa if I were to select a "Sentences starting clutter" where then "prepositional phrases add clutter" would be hidden? Hopefully my question makes sense lol!
<img width="943" alt="kdsjfidsfsdsfjfds" src="https://github.com/freeplane/freeplane/assets/147225942/c5db8770-23d4-47ce-8ede-d7bd10234fb2">

**Comment by rouilj** [comment]
Does the "Filter Selected Nodes" along with "Show Descendents" work?

Show Descendents is on the bottom of the filter rows and looks like a funnel with a connector followed by a node. The Filter Selected nodes is to the right of that and looks like a bunch of nodes stacked on top of a funnel.

**Reply by adamfreeplaneuser01** [reply]
It didn't seem to do anything. Euu2021's idea technically works, but I was hoping that there would be a way to do this more 
"automatically" instead of having to constantly jump in and out

**Comment by euu2021** [comment]
If you want to see just the branch of "Prepositional phrases add clutter", select it and use `Navigate->Jump in (Tab)`.

Then, to go back to full map you can press Esc on the current Root (in that case, "Prepositional phrases add clutter"), or use `Navigate->Jump out`

**Reply by adamfreeplaneuser01** [reply]
Hi! Technically, this idea works, but I was hoping that there would be a way to do this more 
"automatically" instead of having to constantly jump in and out

**Comment by EdoFro** [comment]
I think you mean automatically fold/unfold branches when you navigate the mindmap. is it?

**Reply by adamfreeplaneuser01** [reply]
Yes, that is what I mean. Not sure what you mean by "is it" though

**Comment by euu2021** [comment]
Now, I understand. It's similar to that feature request: https://sourceforge.net/p/freeplane/featurerequests/2545/

If you want to strongly highlight the branch of the currently selected node, automatically, one idea is to use a cloud in the "Selected node" style. The result is this (also, Spotlight is turned on):

![javaw_nWmyalQ4W9](https://github.com/freeplane/freeplane/assets/77707706/47dc1513-412f-4576-9970-ab1596e952ff)

**Comment by i-plasm** [comment]
If I understood your post correctly, this small script would probably do it:
[FilterNodeAncestorsDescendants.zip](https://github.com/freeplane/freeplane/files/12921357/FilterNodeAncestorsDescendants.zip)


The script essentially filters the selected node, making sure the ancestors and descendants are shown. __Note__: this doesn't imply the descendant branch will be fully unfolded. It will be shown exactly as displayed prior to the filtering. Unfolding is a costly operation in general, so I left that out of the script. You can still unfold whenever you need via Freeplane's "Unfold all" action. That said, a script variation can be done to include unfolding the whole branch automatically.

It gets installed on `Filter -> Custom Filters -> Filter Node Ancestors Descendants` . You'd need to assign a shortcut to that script, and also to the `Filter -> No Filtering` option. That way, via your keyboard you'd do the filtering/unfiltering. Note that in order to unfilter you do not need to position yourself on any particular node, so the mechanism is fast.
 
If that's not what you're looking for, then I think @euu2021 's idea of a extremely focused view of the node via tweaking "Selected node" style + using Spotlight is really good and worthy of consideration (see: https://github.com/freeplane/freeplane/discussions/1464#discussioncomment-7266993)
